docs/source/admin/cdni.rst

```diff
@@ -0,0 +1,36 @@
+..
```

Comment:
This document is inaccessible to anyone interacting with the documentation in HTML form, because it's not included in any tables of contents. You'd have to put the URL in manually.